services/company_budget/app/src/Domain/ValueObject/CompanyBudgetId.php

```diff
@@ -0,0 +1,4 @@
+<?php
+namespace App\Domain\ValueObject;
+
+class CompanyBudgetId extends StringValueObject { }
```

Comment:
### Feature Required
Just keep in mind that you need to generate this BudgetId, here or in entity